client/my-sites/checkout/composite-checkout/components/wp-checkout-order-review.tsx

```diff
@@ -227,6 +228,10 @@ function CouponFieldArea( {
 		);
 	}
 
+	if ( isAkismetCheckout() ) {
```

Comment:
I'd recommend moving this check to the render of `WPCheckoutOrderReview`. That way, we avoid calling this function/component entirely

Reply:
If it's moved there, it removes the whole top area where you can interact with what's in the order.

<img width="600" alt="Screenshot 2023-05-31 at 13 14 29" src="https://github.com/Automattic/wp-calypso/assets/4077604/ec1224d1-f45c-4eb1-86d6-ea466d75492d">